subgraphs/osmosis/src/modules/Withdraw.ts

```diff
@@ -0,0 +1,238 @@
+import { Protobuf } from "as-proto";
+import { BigDecimal, BigInt, cosmos, log } from "@graphprotocol/graph-ts";
+import {
+  Withdraw as WithdrawTransaction,
+  LiquidityPool as LiquidityPoolStore,
+} from "../../generated/schema";
+import {
+  MsgExitPool,
+  MsgExitSwapExternAmountOut,
+  MsgExitSwapShareAmountIn,
+} from "./Decoder";
+import * as constants from "../common/constants";
+import { getOrCreateDexAmmProtocol } from "../common/initializer";
+import * as utils from "../common/utils";
+import { updateMetrics } from "./Metrics";
+
+function createWithdrawTransaction(
+  from: string,
+  liquidityPool: LiquidityPoolStore,
+  transaction: cosmos.TxResult,
+  block: cosmos.HeaderOnlyBlock,
+  inputTokenAmounts: Array<BigInt>,
+  sharesBurnt: BigInt,
+  amountUSD: BigDecimal
+): void {
+  if (!transaction) {
+    return;
+  }
+  const withdrawTransactionId = "withdraw-" + transaction.hash.toHexString();

```

Comment:
Id needs log index!

Reply:
Same reason as the comments above for the swap transaction. Thanks!